src/cpp/fastdds/topic/DDSSQLFilter/DDSFilterValue.cpp

```diff
@@ -0,0 +1,356 @@
+// Copyright 2022 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @file DDSFilterValue.cpp
+ */
+
+#include "DDSFilterValue.hpp"
+
+#include <cassert>
+#include <cstring>
+#include <limits>
```

Comment:
This header is not used anymore (it was introduced at some development point where the `std::numeric_limits` were checked in the `compare` method).

Instead the `<string>` and `<regex>` header files should be included.

Reply:
Commit 3d5ece2